data/json/recipes/other/parts.json

```diff
@@ -769,5 +769,20 @@
       [ [ "amplifier", 1 ] ],
       [ [ "cable", 2 ] ]
     ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "BRISK_EXERCISE",
+    "result": "radio_car_wheel",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MATERIALS",
+    "skill_used": "fabrication",
+    "skills_required": [ "fabrication", 6 ],
+    "difficulty": 6,
+    "time": "2 h 30 m",
+    "autolearn": true,
+    "proficiencies": [ { "proficiency": "prof_carving", "time_multiplier": 1.25, "fail_multiplier": 1.25 } ],
```

Comment:
This needs some kind of casting, not just carving.  I'm beyond sceptical that you're going to find rubber chunks large enough to carve tires out of.

If this is supposed to be some kind of really thin disc of solid rubber instead of a typical RC car wheel, it needs to be a new item type to indicate that it's improvised.

Reply:
For simplicity what if I just replace the RC car wheel with 4" set of casters?

Reply:
You could set it so it could have either one